tree_of_thought_llm/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.0"
```

Comment:
again, this could be migrated under a pyproject field

Reply:
Yea I’d usually say have an explicit version file or define it in init but as I said this was just a super simple starter code sorta suggestion. Anyways not sure if you’re a  maintainer or if this is something that’s desired but definitely feel free to modify things if you want

Reply:
I'm not a maintainer, but I was going to offer some similar suggestion on the pyproject to the authors, so I came to join on this effort. :)

On this: I just saw that there was already a `version` field in the pyproject, so I'd propose to remove this duplicate

Reply:
Oh I see, good catch. I think it’s probably best to reach out to the maintainers directly to see if this is something they are interested in, before working more on a PR that may not be necessary from their perspective

Reply:
```suggestion
```